### DISCUSSION TITLE
一段时间后，视频最高清晰度会降至480P

### DISCUSSION THREAD

**Discussion by konata422** [ywmoyue/biliuwp-lite#217] [General]

**Comment by lovepefe** [comment]
同样是这个问题，退出账号 重新登录就好了。但是出现问题很频繁

**Comment by ywmoyue** [comment]
> 同样是这个问题，退出账号 重新登录就好了。但是出现问题很频繁

大概多久出现一次呢

**Comment by lovepefe** [comment]
> > 同样是这个问题，退出账号 重新登录就好了。但是出现问题很频繁
> 
> 大概多久出现一次呢

感觉4-5天。我再用试试 记一下时间

**Comment by konata422** [comment]
> > 同样是这个问题，退出账号 重新登录就好了。但是出现问题很频繁
> 
> 大概多久出现一次呢

<img width="1166" alt="image" src="https://user-images.githubusercontent.com/32641980/230546881-4a26d3fd-b4c0-49be-86c6-6d0e6ae2d235.png">
 出现了

**Comment by konata422** [comment]
> > > 同样是这个问题，退出账号 重新登录就好了。但是出现问题很频繁
> > 
> > 
> > 大概多久出现一次呢
> 
> <img alt="image" width="1166" src="https://user-images.githubusercontent.com/32641980/230546881-4a26d3fd-b4c0-49be-86c6-6d0e6ae2d235.png">
> 
> 出现了

<img width="1167" alt="image" src="https://user-images.githubusercontent.com/32641980/230834774-af987c79-6a3d-414e-9e29-63229465e118.png">
再次出现

**Comment by ywmoyue** [comment]
@hetingrui @lovepefe 问下你们使用的什么方式登录的

**Comment by lovepefe** [comment]
> @hetingrui @lovepefe 问下你们使用的什么方式登录的

扫码 ，哈哈 我现在没复现这个问题。

**Comment by konata422** [comment]
> @hetingrui @lovepefe 问下你们使用的什么方式登录的

扫码
而且我今天一天就遇见了两次

**Comment by Landidi666** [comment]
在我这里是把「优先视频编码」改成H265和AV1就会出现480P的情况，即使重新登录，一段时间后也会变成480P。

使用H264暂时没发现（也可能是使用H264解码变成480P需要更长的等待时间，目前30分钟内测试正常），或许是由于H265和AV1视频源的请求方式不对？

**Comment by ywmoyue** [comment]
可能与cookie刷新机制有关 https://github.com/SocialSisterYi/bilibili-API-collect/blob/a46f140fde73530868ce9db5f613deed8d5c645e/docs/login/cookie_refresh.md

我有空研究下

**Comment by ywmoyue** [comment]
4.6.3 已添加了Cookie刷新机制，麻烦测试一下看还会不会出现已登录时视频最高清晰度只有480P的情况 @hetingrui @lovepefe @Landidi666

**Comment by ywmoyue** [comment]
超过一个月无人回复，当前issue转为discussion

**Reply by ywmoyue** [reply]
问题之前版本已解决，此discussion关闭
